golang/cosmos/x/vbank/vbank_test.go

```diff
@@ -730,16 +746,32 @@ func Test_EndBlock_Rewards(t *testing.T) {
 	}
 }
 
-type mockAccount struct{}
+type mockAccount struct{
```

Comment:
Please name "mockAccountKeeper" or "mockAuthKeeper". (Yes, I know the misleading name predates this PR.)